draft-ietf-quic-transport.md

```diff
@@ -594,7 +594,9 @@ of subsequent packets that it sends.
 A Retry packet does not include a packet number and cannot be explictly
 acknowledged by a client.
 
-A server MUST only send a Retry in response to a client Initial packet.
+A server MUST NOT send a Retry in response to packets other than Initial
+or 0-RTT packets.  A server MAY discard 0-RTT packets and only send Retry in
+response to Initial packets.
```

Comment:
I would invert this last sentence:

"A server MAY choose to send Retry in response to Initial packets and discard or buffer 0-RTT packets corresponding to unvalidated client addresses."

Reply:
Yes, that it better.  Thanks.